modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,111 @@
-= Creating a custom ConfigMap for importing a virtual machine
+= Creating a custom ConfigMap
 
-If the operating system of the Red Hat Virtualization (RHV) virtual machine does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap to map the RHV operating system to a common {VirtProductName} template. The template is used to create the new virtual machine.
+You can create a custom ConfigMap for a Red Hat Virtualization (RHV) virtual machine if its operating system does not have a corresponding {product-title} template.
 
 [NOTE]
 ====
-In the current release, you must create a Windows template and a custom ConfigMap to import a RHV VM with a Windows operating system. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ#1856654] for details.
+If you are importing a Windows virtual machine, you must also create a Windows template. link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[(*BZ#1856654*)]
 ====
 
 .Procedure
 
-. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` as in the following example:
+. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>`. The operating system appears in the `<os>` section of the XML output, as shown in the following example:
 +
 [source,xml]
 ----
 <os>
 ...
-<type>windows_2016x64</type>
+<type>rhel_8x64</type>
 </os>
 ----
 
-. Obtain a list of the common {VirtProductName} operating system templates by running the following command on your {product-title} cluster:
+. To import a Linux VM:
+
+.. Obtain a list of the {product-title} templates:
 +
 ----
 $ oc get templates -n openshift --show-labels | tr ',' '\n' | grep os.template.kubevirt.io | sed -r 's#os.template.kubevirt.io/(.*)=.*#\1#g' | sort -u
 ----
 +
-If the RHV VM operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap.
+The output is similar to the following example:
++
+----
+fedora31
+fedora32
+...
+rhel8.1
+rhel8.2
+...
+----
++
+If the RHV VM operating system appears in this template list, you do not need to create a ConfigMap.
+
+.. If the RHV VM operating system does not appear in the template list, create a ConfigMap for a common template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default <1>
+data:
+  guestos2common: |
+    "Red Hat Enterprise Linux Server": "rhel" <2>
+    "CentOS Linux": "centos"
+    "Fedora": "fedora"
+    "Ubuntu": "ubuntu"
+    "openSUSE": "opensuse"
+  osinfo2common: |
+    "<operating-system>": "<common-template>" <3>
+EOF
+----
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify the Linux operating system.
+<3> Specify the Linux operating system version and the {product-title} common template that it maps to:
```

Comment:
This is the Linux section. They will be mapping to a CNV common template. They only map Windows to a custom template.

Reply:
No. This is not a "Linux" section. Any mapping can added here. Windows too.

so it can be something like:
Specify the VM operating system, as it appears in RHV Rest api, and the {product-title} common template that it maps to